department-of-veterans-affairs/caseflow
# Vinner57/appeals 29192

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/APPEALS-29192)

# Description
Created a migration and a model for Vbms Document Types

## Acceptance Criteria

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below

1. For feature branches merging into master: Was this deployed to UAT?
2. Run rake db:migrate to create the new table
3. Run the following command to populate the data in to the new table:
4. bundle exec rails runner scripts/run_seed_data_file.rb db/seeds/vbms_document_types.rb
5. Go into the backend by running bundle exec rails c.
6. Check that we have records inside the vbms_document_types table by running VbmsDocumentType.count and see 
    the output shows 1 record.


# Frontend
## User Facing Changes

<img width="856" alt="Screen Shot 2023-11-07 at 8 49 04 AM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/128258952/6c34134a-44d7-4ff9-a704-60e690293e74">

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
